packages/styles/src/components/list-group.scss

```diff
@@ -1,35 +1,21 @@
-@forward './../variables/options';
-
-@use 'sass:map';
-
 @use './../themes/bootstrap/core' as *;
-@use './../themes/bootstrap/list-group' as bl;
 
-@use './../variables/color';
-@use './../variables/spacing';
-@use './../variables/type';
-@use './../variables/commons';
 @use './../variables/animation';
-@use './../variables/components/list-group';
-@use './../mixins/type' as type-mixins;
+@use './../mixins/utilities' as utilities-mx;
+
+@use '../functions/tokens';
+@use '../tokens/components';
+
+tokens.$default-map: components.$post-list;
 
 // Base class
 .list-group {
   background-color: transparent;
-
-  &-box {
-    padding-top: map.get(spacing.$post-sizes, 'mini');
-    padding-bottom: map.get(spacing.$post-sizes, 'mini');
-    box-shadow: commons.$box-shadow;
-  }
+  list-style: none;
+  padding: 0;
 }
 
 .list-group-item-action {
-  @include transition(
-    color animation.$transition-time-simple animation.$transition-easing-default,
-    background-color animation.$transition-time-simple animation.$transition-easing-default
-  );
-
   &:hover,
   &:focus {
     .list-group-item-right-svg {
```

Comment:
Would be nice if we could omit the class here: `> post-icon:last-child {...}`

Reply:
I targeted only the post-icon of the list-group-link instead of using :last-child